config-controller/api/v1alpha1/policy_types.go

```diff
@@ -66,7 +72,7 @@ type SecurityPolicySpec struct {
 	Severity string `json:"severity"`
 	// Enforcement lists the enforcement actions to take when a violation from this policy is identified.  Possible value are UNSET_ENFORCEMENT, SCALE_TO_ZERO_ENFORCEMENT, UNSATISFIABLE_NODE_CONSTRAINT_ENFORCEMENT, KILL_POD_ENFORCEMENT, FAIL_BUILD_ENFORCEMENT, FAIL_KUBE_REQUEST_ENFORCEMENT, FAIL_DEPLOYMENT_CREATE_ENFORCEMENT, and. FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT.
 	EnforcementActions []EnforcementAction `json:"enforcementActions,omitempty"`
-	// Notifiers is a list of IDs of the notifiers that should be triggered when a violation from this policy is identified.  IDs should be in the form of a UUID and are found through the Central API.
+	// Notifiers is a list of IDs or names of the notifiers that should be triggered when a violation from this policy is identified.  IDs should be in the form of a UUID and are found through the Central API.
 	Notifiers []string `json:"notifiers,omitempty"`
```

Comment:
I like making this field dynamically support both UUIDs and names, but have we considered adding a separate field like `notifierNames`?  The current approach matches the UX in the UI but it does diverge from the API.  Just want to make sure we've considered both options.

Reply:
How does it diverge from the API? The API accepts an array of strings.

Reply:
But if that string is not a UUID, the API rejects it, unlike the behavior in this PR.

Reply:
Yes, the CRD mimics the UI. Another way of interacting with our API in its current form. I will keep as is, and look into changing the API.